.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+Thank you for submitting the pull request to the Apache Ignite.
+
+In order to streamline the review of the contribution 
+we ask you to ensure the following steps have been taken:
+
+### The Contribution Checklist
+- [ ] There is a single JIRA ticket related to the pull request. 
+- [ ] The web-link to the pull request attached to the JIRA ticket.
+- [ ] The JIRA ticket has the _Patch Available_ state.
+- [ ] The pull request body describes changes that have been made. 
+The description explains _WHAT_ and _WHY_ was made instead of _HOW_.
+- [ ] The pull request title treated as the final commit message. 
+The following pattern must be used: `IGNITE-12407: Java thin client. Cluster API support`
```

Comment:
Subject should have imperative mood: `IGNITE-12407: Add Cluster API support to Java thin client`, or `IGNITE-12407: Java thin client - add Cluster API support`

Reply:
Fixed.